app/assets/stylesheets/account/contact.scss

```diff
@@ -0,0 +1,99 @@
+#contact_header {
+  margin-bottom: 30px;
+  border-bottom: 1px solid #E6E6E6;
+}
+
+form#contact_search {
+  position: relative;
+  margin: 20px 0;
+  div.field {
+    display: inline-block;
+    label {
+      padding-right: 10px;
+    }
+    input {
+      margin-right: 10px;
+      width: 180px;
+    }
+  }
+  div.actions {
+    position: absolute;
+    right: 50px;
+    display: inline-block;
+    input {
+      height: 35px;
+      border: 1px solid #0a3c59;
+      background: #3e779d;
+      padding: 0px 21px;
+      border-radius: 6px;
+      text-shadow: #7ea4bd 0 1px 0;
+      color: #e9e3e3;
+      font-size: 14px;
+      text-decoration: none;
+      vertical-align: middle;
+      background-image: linear-gradient(#65a9d7 0%, #3e779d 100%);
```

Comment:
I don't understand why buttons/inputs must be different on this page from others?
Let's be consistent. You can use some styles from theme BUT all inputs/buttons must look similar across whole app